test.js

```diff
@@ -48,4 +48,29 @@ describe('csslint-stylish', () => {
 
     assert(CSSLint.hasFormat('stylish'), 'csslint should be stylish')
   })
+
+  it('should be able to report from CSSLint without undefined line output', () => {
```

Comment:
Line is too long.<br>'arrow function syntax (=>)' is only available in ES6 (use esnext option).